<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlWidget</class>
 <widget class="QWidget" name="ControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>140</width>
    <height>426</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>140</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>140</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>140</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QPushButton" name="peace_btn">
     <property name="minimumSize">
      <size>
       <width>88</width>
       <height>132</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>88</width>
       <height>132</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>88</width>
       <height>132</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/chess/image/button/draw_nor.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="surrender_btn">
     <property name="minimumSize">
      <size>
       <width>88</width>
       <height>132</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>88</width>
       <height>132</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>88</width>
       <height>132</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/chess/image/button/surrender_nor.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="undo_btn">
     <property name="minimumSize">
      <size>
       <width>88</width>
       <height>132</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>88</width>
       <height>132</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>88</width>
       <height>132</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/chess/image/button/undo_nor.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
